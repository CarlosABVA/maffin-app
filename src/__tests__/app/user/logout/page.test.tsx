import React from 'react';
import { render } from '@testing-library/react';

import LogoutPage from '@/app/user/logout/page';

describe('LogoutPage', () => {
  it('matches snapshot', () => {
    const { container } = render(<LogoutPage />);
    expect(container).toMatchSnapshot();
  });

  it('sets empty accessToken', () => {
    localStorage.setItem('accessToken', 'token');
    render(<LogoutPage />);

    expect(localStorage.getItem('accessToken')).toEqual('');
  });
});
