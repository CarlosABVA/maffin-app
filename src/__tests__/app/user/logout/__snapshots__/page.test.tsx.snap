// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoutPage matches snapshot 1`] = `
<div>
  <div
    class="text-center"
  >
    <p>
      See you soon!
    </p>
    <a
      href="/user/login"
    >
      <b>
        Log In again
      </b>
    </a>
  </div>
</div>
`;
